.travis.yml

```diff
@@ -2,14 +2,12 @@ language: java
 
 sudo: false
 
-script:
-  - jdk_switcher use oraclejdk7
-  - mvn clean verify -P jdk7
-  - jdk_switcher use oraclejdk8
-  - mvn clean verify -P jdk8
+jdk:
+  - oraclejdk8
+  - oraclejdk7
 
-after_success:
-  - mvn clean test jacoco:report coveralls:report
+script:
```

Comment:
> If you click on *Show all checks" you should be able to see all three results.

Yes, but with the jacoco check running as part of the main build I would see a failing build and have to look in the travis output to see whether it failed because of a test or not enough coverage.

I just noticed we have the jacoco goals always bound to the `verify` phase, so the separate `jacoco:report` on the command line should not be needed.